### ISSUE TITLE
Update the status of each module as it completes

### ISSUE THREAD

**Issue by rmlockwood** [cdfarrow/flextools#10]
If I run multiple modules at once I would like to see the status of each one as it completes. See this video which shows the problem. 
https://www.loom.com/share/892958a307874d0ca94a4e3b762ea865

**Comment by cdfarrow** [comment]
Yes, I've been aware of this since the beginning. I think it is because everything is running in a tight loop and the UI never gets a look-in to refresh itself. It looks like the processing should be put on another thread or use BackgroundWorker:
https://stackoverflow.com/a/573270
https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.backgroundworker?view=net-7.0

**Comment by cdfarrow** [comment]
May need coordination with #47.

**Comment by cdfarrow** [comment]
Fixed with regular calls to DoEvents().
